third_party/gfootball_engine/src/systems/graphics/rendering/opengl_renderer3d.cpp

```diff
@@ -384,10 +390,12 @@ bool OpenGLRenderer3D::CreateContext(int width, int height, int bpp,
 
   SDL_GL_SetAttribute(SDL_GL_ACCELERATED_VISUAL, 1);  // DISABLED?
 
-  // SDL_GL_SetAttribute(SDL_GL_CONTEXT_MAJOR_VERSION, 3);
-  // SDL_GL_SetAttribute(SDL_GL_CONTEXT_MINOR_VERSION, 2);
-  // SDL_GL_SetAttribute(SDL_GL_CONTEXT_PROFILE_MASK,
-  // SDL_GL_CONTEXT_PROFILE_CORE);
+  // https://stackoverflow.com/questions/22158542/specify-opengl-version-using-sdl2 
+  #ifdef __APPLE__
+  SDL_GL_SetAttribute(SDL_GL_CONTEXT_PROFILE_MASK, SDL_GL_CONTEXT_PROFILE_CORE);
```

Comment:
This does not have any effect and I think is because at this point it's already too late to request a more recent OpenGL version